with open('/Users/congshi/PycharmProjects/DHTutorial/templates/search/indexes/blog/note_text.txt', encoding='utf-8') as fp:
    print(fp.read())

# print('/Users/congshi/PycharmProjects/DHTutorial/templates/search/indexes/blog/note_text.txt' == '/Users/congshi/PycharmProjects/DHTutorial/templates/search/indexes/blog/note_text.txt')

!!!!

What is the fuck ? The file exist, but python code wrong.